pkg/kubectl/cmd/util/helpers.go

```diff
@@ -401,8 +401,8 @@ func AddDryRunFlag(cmd *cobra.Command) {
 }
 
 func AddServerSideApplyFlags(cmd *cobra.Command) {
-	cmd.Flags().Bool("server-side", false, "If true, apply runs in the server instead of the client.")
-	cmd.Flags().Bool("force-conflicts", false, "If true, server-side apply will force the changes against conflicts.")
+	cmd.Flags().Bool("server-side", false, "ALPHA -- If true, apply runs in the server instead of the client.")
```

Comment:
Change this to `This is an alpha feature and flag` similarly to how we do it in https://github.com/kubernetes/kubernetes/blob/00eab3c40bcda9ee73eac060242d36602e627fec/pkg/kubectl/cmd/apply/apply.go#L179